### ISSUE TITLE
Providing sharedObjectPaths doesn't have any effect

### ISSUE THREAD

**Issue by pbajurko** [bugsnag/bugsnag-android-gradle-plugin#293]
### Describe the bug
Plugin does not process native symbol files anymore. 
Looks like plugin's configuration variable is not used anywhere see https://github.com/bugsnag/bugsnag-android-gradle-plugin/search?q=sharedObjectPaths&unscoped_q=sharedObjectPaths

```
uploadBugsnagNdk-releaseMapping
Starting ndk upload
Processing shared object files

```

### Steps to reproduce
Integrate plugin in version 5.x.x and follow [UPGRADING.md](https://github.com/bugsnag/bugsnag-android-gradle-plugin/blob/master/UPGRADING.md)

```
def paths = [
    new File("app/build/jni/libs"),
    new File("app/build/someOtherFolder")
]
bugsnag {
    sharedObjectPaths = paths
}

```

**Comment by johnkiely1** [comment]
Hi @pbajurko 

Thank for reporting this. We are looking into creating a fix for this and aim to include it in the next release.

**Comment by steve-nester-uk** [comment]
Fixed in [v5.1.0](https://github.com/bugsnag/bugsnag-android-gradle-plugin/releases/tag/v5.1.0)
